ui/admin/app/templates/scopes/scope/targets/index.hbs

```diff
@@ -37,37 +54,74 @@
         </table.header>
         <table.body as |body|>
           {{#each @model as |target|}}
-            <body.row as |row|>
-              <row.headerCell>
-                {{#if (can 'read model' target)}}
-                  <LinkTo
-                    @route='scopes.scope.targets.target'
-                    @model={{target.id}}
-                  >
+            {{#if (feature-flag 'ssh-target')}}
+
+              <body.row as |row|>
+                <row.headerCell>
+                  {{#if (can 'read model' target)}}
+                    <LinkTo
+                      @route='scopes.scope.targets.target'
+                      @model={{target.id}}
+                    >
+                      {{target.displayName}}
+                    </LinkTo>
+                  {{else}}
                     {{target.displayName}}
-                  </LinkTo>
-                {{else}}
-                  {{target.displayName}}
-                {{/if}}
-                <p>{{target.description}}</p>
-              </row.headerCell>
-              <row.cell>
-                {{#if target.type}}
-                  <Rose::Badge>
-                    {{t (concat 'resources.target.types.' target.type)}}
-                  </Rose::Badge>
-                {{/if}}
-              </row.cell>
-              <row.cell>
-                <Copyable
-                  @text={{target.id}}
-                  @buttonText={{t 'actions.copy-to-clipboard'}}
-                  @acknowledgeText={{t 'states.copied'}}
-                >
-                  <code>{{target.id}}</code>
-                </Copyable>
-              </row.cell>
-            </body.row>
+                  {{/if}}
+                  <p>{{target.description}}</p>
+                </row.headerCell>
+                <row.cell>
+                  {{#if target.type}}
+                    <Rose::Badge>
+                      {{t (concat 'resources.target.types.' target.type)}}
+                    </Rose::Badge>
+                  {{/if}}
+                </row.cell>
+                <row.cell>
+                  <Copyable
+                    @text={{target.id}}
+                    @buttonText={{t 'actions.copy-to-clipboard'}}
+                    @acknowledgeText={{t 'states.copied'}}
+                  >
+                    <code>{{target.id}}</code>
+                  </Copyable>
+                </row.cell>
+              </body.row>
+            {{else}}
+              {{#if (eq target.type 'tcp')}}
```

Comment:
If `feature-flag 'ssh-target'` is false, and `target.type` is not equal to `tcp`. What we will show here? and empty cell?

Reply:
yeah, we don't want to show ssh type if the `feature-flag 'ssh-target` is set to false. So it will be an empty cell if there are no TCP type.

Reply:
Do we have an `isTCP` computed property?  We generally prefer them over equality checks in the template.

Reply:
Gotcha :ok_hand:

Reply:
@randallmorey good idea! I'll check that.

Reply:
updated!